docs/source/theory/boundaryConditions.rst

```diff
@@ -943,38 +943,44 @@ field values are used for property evaluations. When flow is leaving the
+
+.. _theory_weak_symmetry_bc:
+
+Weak Symmetry Boundary Condition
 +++++++++++++++++++++++++++
 
 Continuity, Mixture Fraction, Enthalpy, Species, :math:`k_{sgs}`, k and :math:`\omega`
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Zero diffusion is applied at the symmetry bc.
+Weak symmetry applies zero diffusion at the boundary for scalar quantities, which effectively sets the boundary-normal gradients of these quantities to zero. This means that, unlike for strong symmetry, in the weak symmetry implementation, normal velocity can be non-zero. This is possible because the cell averaged quantities in Nalu-Wind's discretization are stored at the nodes and therefore reside on the boundaries. In general, a non-zero normal velocity can cause net inflow or outflow. The Poisson solve prevents this by enforcing mass conservation globally to the order of linear solver convergence. 
 
 Momentum
-~~~~~~~~
+~~~~~~~
 
-A symmetry boundary is one that is described by removal of the
-tangential stress. Therefore, only the normal component of the stress is
-applied:
+A symmetry boundary is one that is described by removal of the tangential stress. For weak symmetry this is done in the momentum equation by applying only the normal component of stress:
 
 .. math:: F^n_x = (F_x n_x + F_y n_y ) n_x,
 
 which can be written in general component form as,
 
 .. math:: F^n_i = F_j n_j n_i.
 
-Strong versus Weak Symmetry
-~~~~~~~~~~~~~~~
+The momentum equation also penalizes non-zero normal velocity. The strength of this penality depends on the penalty factor. One can enforce a stronger representation of the boundary by amplifying the penalty factor through the `symmetry_bc_penalty_factor` variable in `solution_options`. Its default value is 2.0 which is the minimum required for stability. Amplifying the penalty factor will enforce the boundary in a stronger sense, but can also lead to a more difficult matrix solve as with any large penalty term.
+
+Both strong and weak symmetry boundary conditions have associated error. In the weak form the error manifests as non-zero local velocities at the boundary. As the mesh is refined, the weak symmetry boundary condition converges to the strong symmetry boundary condition (zero normal velocity) with a first order rate. Note that this doesn't mean the error goes away; the error just moves off the boundary and into the domain.
```

Comment:
```suggestion
Both strong and weak symmetry boundary conditions have associated error. In the weak form the error manifests as non-zero local velocities at the boundary. As the mesh is refined, the weak symmetry boundary condition converges to the strong symmetry boundary condition (zero normal velocity) with a first order rate. 
```
The error _does_ go away under mesh refinement for both the strong and weak formulations of the boundary condition, but it only decreases with first order rate.  You can eventually get to DNS.  The main point in commenting on error is that switching from weak to strong does not make the boundary representation's error disappear.

Reply:
Changed the last line to include the additional info. from your description here (rather than removing the last line.)